app/controllers/api/v1/departments_controller.rb

```diff
@@ -0,0 +1,49 @@
+class Api::V1::DepartmentsController < ApplicationController
+  before_action :find_department, only: [:show, :update, :destroy, :employees]
+  # decorates_assigned :department
+
+  def index
+    @departments = Department.all
+  end
+
+  def show
+    employees = @department.employees
+    @current_manager = Department.current_manager
```

Comment:
`Department.current_manager(department_id)` because you have to fetch for a special department